students/grzegorz_bajorski/lesson_10_organizing_files/deleting_inneeded_files.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+max_size = 1024 * 1024 * 100
+
+for filename in os.listdir():
```

Comment:
```os.listdir()``` should be used with path argument. In a case when you want to list the current directory it should look:
```os.listdir('.')```